nixos/roles/mysql.nix

```diff
@@ -116,6 +116,20 @@ with builtins;
       }
     ];
 
+    systemd.tmpfiles.rules = [
+      "d /var/log/mysql 0755 mysql mysql -"
+      "f /var/log/mysql/mysql.slow 0644 mysql mysql -"
```

Comment:
The file shouldn't be readable by everybody, just service users, I suggest 0640 mysql service